demo/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import include, path, re_path
+
+from config import urls as root_urls
+from demo import views
+
+
+urlpatterns = [
+    path('api/demo/v0/load_data/', views.load_data, name='load_data'),
+    path('api/demo/v0/datasets_exist/', views.data_exists, name='datasets_exist'),
+    path('<db_name>/', views.SchemasView.as_view(), name='schemas'),
+    re_path(
```

Comment:
This is a wildcard regex it would end up overriding all the URLs, so for any URL other than `api/demo`, `SchemaView` will be called. For example, calling `/auth/login/` will go to `SchemasView` instead of the Django login view. We would have to split the `SchemasView` regex in the mathesar URL into a separate path so that other apps can add extend the API paths without extending the `SchemasView` path